include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -470,6 +470,20 @@ class FEPointEvaluation
   reinit(const typename Triangulation<dim, spacedim>::cell_iterator &cell,
          const ArrayView<const Point<dim>> &unit_points);
 
+  /**
+   * Reinitialize the evaluator to point to the correct precomputed mapping of
+   * the cell in the MappingInfo object.
+   */
+  void
+  reinit(const unsigned int cell_index);
+
+  /**
+   * Reinitialize the evaluator to point to the correct precomputed mapping of
+   * the face in the MappingInfo object.
+   */
+  void
+  reinit(const unsigned int cell_index, const unsigned int face_number);
```

Comment:
Can we keep the name general? `cell_index` and `face_index` is quite specific. These are rather indices into the `MappingInfo`.

Reply:
What would you suggest?